wordpress-mobile/WordPress-Android
# Update app icons to spectrum background

This PR updates the app's icons to use the new spectrum background:

<img src="https://user-images.githubusercontent.com/830056/121730334-ec7e4b80-cac5-11eb-8bb0-89bb8cf4bf9d.png" width=500/>

#### To test:

Build the vanilla variant of the app and verify that the icon has the new background. Try it out with different screen resolutions and Android versions.

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
